Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -45,6 +46,7 @@ noise::module::ModulePtr TilePartitionerNoise::makeCornerModule(const Corners& c
     combiner.setInputModuleSource(2, central.getOutputModule());
     if (mDebugOutput)
     {
+        setLoggingLevel(logging::Level::Debug);
```

Comment:
Logging level and append mode should be set by the user either in command line options or in the JSON config file, and this change to the logging level should be undone before leaving this code block.

Also, there should be a comment explaining why the logging level is changed. I guess it's because the messages below are required to prompt user interaction.